package.json

```diff
@@ -76,5 +77,8 @@
   "workspaces": [
     "./lib/*",
     "./sandbox"
-  ]
+  ],
+  "engines": {
+    "node": ">16"
+  }
```

Comment:
The only LTS versions of Node since 16 were released 4 months ago. Are we using Node features that aren't available in v16 and lower?
https://nodejs.org/en/download/releases/
I'm not opposed to this change, I just don't want to risk alienating anyone who may be reluctant to upgrade Node.